tests/testthat/test-loading-data.R

```diff
@@ -91,7 +91,7 @@ test_that('PLS recipe with no options', {
 test_that('PLS workflow with no options', {
     raw_pls <-
         pls_workflow %>%
-        pull_workflow_prepped_recipe() %>%
+        extract_recipe() %>%
```

Comment:
We'll probably need a version constraint in the description file to use these (I'm in favor).